soapui-installer/src/dist_standalone/bin/installationcomplete.bat

```diff
@@ -1,2 +0,0 @@
-set CLASSPATH=..\lib\*
-..\jre\bin\java -cp "%CLASSPATH%" TrackOutAppAction InstallSoftware Product SoapUIOS Version %1 success true
```

Comment:
changes like removing installationcomplete/uninstallationcomplete/updateinstallation.(bat/sh) won't be applied, please revert it.

Reply:
I will. Actually, I touched those by accident :/

Reply:
No changes to installationcomplete/uninstallationcomplete/updateinstallation anymore